.github/workflows/main.yml

```diff
@@ -0,0 +1,58 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    name: test-and-build ${{ matrix.target_os }}_${{ matrix.target_arch }}
+    runs-on: ${{ matrix.os }}
+    container: weaveworks/eksctl-build:63bb98c9599ef0b0cc8bc08f64b6d572765e160e
```

Comment:
I don't like the fact that image hash is hard coded here, but as per the flow, docker pull is running before source checkout step :cry: